common/djangoapps/student/views/dashboard.py

```diff
@@ -629,6 +632,20 @@ def student_dashboard(request):
 
     enterprise_message = get_dashboard_consent_notification(request, user, course_enrollments)
 
+    recovery_email_message = None
+
+    # Update the tag info with 'enterprise_learner' if the user belongs to an enterprise customer.
+    does_secondary_email_exist = AccountRecovery.objects.filter(user=user).exists()
+    if is_secondary_email_feature_enabled_for_user(user=user) and not does_secondary_email_exist:
+        recovery_email_message = Text(_("Add a recovery email to retain access when single-sign on is not available. "
```

Comment:
```python
if is_secondary_email_feature_enabled_for_user(user=user) and not secondary_email_exists:
    recovery_email_message = Text(
        _(
            'Add secondary email to retain access when single-sign on is not available. '
            'Go to {link_start}your account settings{link_end} to add secondary eamil address.'
        )
    ).format(
        link_start=HTML("<a target='_blank' href='{account_setting_page}'>").format(
            account_setting_page=reverse('account_settings'),
        ),
        link_end=HTML("</a>")
    )
```

Reply:
I will update the indentation but Text was provided by UX team.